### DISCUSSION TITLE
Exclude unnecessary layers for manufacturing.

### DISCUSSION THREAD

**Discussion by boromyr** [bennymeg/Fabrication-Toolkit#165] [Q&A]
How can I exclude unnecessary layers, such as F.Fab?

**Comment by bennymeg** [comment]
Currently, you can remove layers only manually.

**Comment by boromyr** [comment]
But why were all the layers added even if they are unnecessary? Alternatively, can you tell me which function extracts the layers and I will modify it myself? Thank you.

**Comment by bennymeg** [comment]
Only the bare minimum layers are included by default.
Maybe you activated all the layers by mistake somehow.

**Comment by boromyr** [comment]
Same project, I went back to version 4.0 and the exported levels are about half of those with the latest version, I haven't changed anything.
![image](https://github.com/bennymeg/Fabrication-Toolkit/assets/52782032/eeaa1fb1-90af-4fd2-9db4-d9c9ee47d97e)
![image](https://github.com/bennymeg/Fabrication-Toolkit/assets/52782032/e832fdee-55a6-4540-9d33-3d976eab1ff6)

**Comment by gmarocco** [comment]
I have the same issue with the release 4.4.1. Even if I disbale the visualization of non manufacturing layers (for example user1) , the corresponding gerber files are generated. Another question as I don't find a clear explanation: what is the option "Apply automatic translations?"
